### ISSUE TITLE
[Bug] Latest update breaks the theme

### ISSUE THREAD

**Issue by MarioVillani** [Laravel-Backpack/theme-tabler#162]
# Bug report

### What I did

Install latest theme tabler version

### What I expected to happen

Theme working

### What happened

local.ERROR: Undefined variable $withColumns after logging in in Backpack dashboard.
#1 /home/myproject/vendor/backpack/theme-tabler/resources/views/components/menu-dropdown.blade.php(43): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->Illuminate\\Foundation\\Bootstrap\\{closure}()

### Context to replicate the bug - Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

```
### LARAVEL VERSION:
10.45.1.0

### BACKPACK PACKAGE VERSIONS:
backpack/backupmanager: v5.0.0
backpack/basset: 1.2.4
backpack/crud: 6.6.3
backpack/generators: v4.0.3
backpack/logmanager: v5.0.1
backpack/permissionmanager: 7.1.1
backpack/pro: dev-main
backpack/revise-operation: 2.0.0
backpack/theme-tabler: 1.2.1
```

**Comment by siberfx** [comment]
**$withColumns** parameter in menu-dropdown.blade.php is a parameter is not defined in component file himself

**Comment by MarioVillani** [comment]
I think @pxpm fixed that in d124fcb 👍

**Comment by pxpm** [comment]
Thanks and sorry guys for the bad experience. 

Just pushed the fix in 1.2.2. 

I didn't wanted to bump crud version and forgot to add the check if the variable is set 🤷 

I will add docs too about it tomorrow, basically allows you to create menus like in tabler demo, side by side: 
![image](https://github.com/Laravel-Backpack/theme-tabler/assets/7188159/0d183f20-514f-4715-bf18-680297d8f644)

You will still need to update CRUD if you want to use this new functionality as you saw by this error. 

Cheers
